mesa/experimental/cell_space/cell.py

```diff
@@ -69,6 +71,7 @@ def __init__(
         self.capacity: int = capacity
         self.properties: dict[Coordinate, object] = {}
         self.random = random
+        self.property_layers: dict[str, PropertyLayer] = {}
```

Comment:
sorry I don't understanand why this is here? should this not be linked to the property layers defined at the grid level?

Reply:
It also is, but it might be useful to be able to access your value in a PropertyLayer directly from the cell.

Reply:
I understand that and agree with it. I now also looked at `add_property_layer` and start to understand what is going on. 

1. I suggest making this `self._property_layers` so it's not declared as part of the public API. My concern is that one might add a layer via cell directly which would then not exist on the grid. 
2. I think overall, a more elegant solution is possible here (but that can be a separate PR). My idea would be to add on the fly Descriptors to cells for each property layer. This would make it possible to de `cell.property_layer_name`, which would return the value for this cell in the named property layer. Another benefit would be that we don't have code by default in cell that is only functional in OrthogonalGrids.

Reply:
On 2) my high-level idea would be that we use the existing property dictionary of the cell and add to it an entry with the name of the property layer, so `cell.properties[property_layer_name]`. *getting* the value should be easy, but I don't know how that would work for setting the value. Any ideas?

Reply:
I can't think of an immediate solution other than to subclass dict and have a dedicated PropertiesDict for all this.

Reply:
Was thinking about that. It might be great as an API, but can also create naming conflicts.

Unfortunately, I'm really making a sprint on my thesis now, so not much conceptual-thought capacity to work on this.

Reply:
With the one minor change (to a private attribute), I think we can merge this for now. I might have some time over the weekend to play around with more descriptor fun 😄 and see what I can do.

Reply:
Go ahead in making the change and merging!